components/dashboard/components/CourseConnections/CourseConnectionsView.jsx

```diff
@@ -210,9 +210,17 @@ class CourseConnectionsView extends React.Component {
      * Draw the graph w/ the latest interaction data
      */
     drawGraph() {
-        const {userInteractions, userLearningGroups} = this.props;
+        const {userLearningGroups} = this.props;
```

Comment:
imo it's cleaner to just directly reference `this.props.userLearningGroups` when setting this variable instead of destructuring now that it's just the one

Reply:
Yeah agreed. I guess I was leaving it for later, but we can remove it for now.

Reply:
Changed.

Reply:
:+1: